draft-ietf-httpbis-http2bis.xml

```diff
@@ -3297,10 +3297,12 @@ cookie: e=f
             format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref> setting to limit the number of
             responses that can be concurrently pushed by a server. Advertising a <xref
             target="SETTINGS_MAX_CONCURRENT_STREAMS"
-            format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref> value of zero disables server push
-            by preventing the server from creating the necessary streams. This does not prohibit a
-            server from sending <xref target="PUSH_PROMISE" format="none">PUSH_PROMISE</xref>
-            frames; clients need to reset any promised streams that are not wanted.
+            format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref> value of zero prevents the server
+            from opening the streams necessary to push responses. However, this does not prevent the
+            server from reserving streams using <xref target="PUSH_PROMISE"
+            format="none">PUSH_PROMISE</xref> frames, because "reserved" streams do not count toward
+            the concurrent stream limit. Clients that do not wish to receive pushed resources need
+            to reset any unwanted reserved streams.
```

Comment:
Should we explicitly point towards `SETTINGS_ENABLE_PUSH` here? It's only a few paragraphs away in the text but this is the second time we've had a review that missed that setting, and it fits well with the text here:

> Clients that do not wish to receive pushed resources need to reset any unwanted reserved streams, or set `SETTINGS_ENABLE_PUSH` to `0` [xref whatever].